### PR TITLE
change colors to presets, various housekeeping

### PR DESCRIPTION
**Pull request by MisterChocker**
Fixes #[issue_no]

## Summary of Changes

- some housekeeping
- Amber is now amber,
- Cyan is now cyan,
- Red is now red,
- Made yellow messages amber.
- Simplified message colors.
- Added missing green to PERF PAGE TO RWY FIELD and PROG PAGE ECON CRZ
- Added missing sub row on airways page
- Added new flight phases to PROG PAGE (preflight, taxi -> having same functionality as takeoff)

## Screenshots (if necessary)

Before:
image

After:
image

## References

image
image

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js [hunk 1]
```diff
@@ -204,7 +204,7 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
 
     checkDestData() {
         if (!isFinite(this.perfApprQNH) || !isFinite(this.perfApprTemp) || !isFinite(this.perfApprWindHeading) || !isFinite(this.perfApprWindSpeed)) {
-            this.addTypeTwoMessage("ENTER DEST DATA", "#ffff00", () => {}, () => {
+            this.addTypeTwoMessage("ENTER DEST DATA", "yellow", () => {}, () => {
```
Comment (beheh) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js:207]: This should be amber ("red"), not yellow:
```suggestion
            this.addTypeTwoMessage("ENTER DEST DATA", "red", () => {}, () => {
```

Documentation:

image

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js [hunk 2]
```diff
@@ -338,7 +338,7 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
                     SimVar.SetSimVarValue("L:A32NX_GPS_PRIMARY_LOST_MSG", "Bool", 1);
                     SimVar.SetSimVarValue("L:GPSPrimaryMessageDisplayed", "Bool", 0);
                     this.tryRemoveMessage("GPS PRIMARY");
-                    this.addTypeTwoMessage("GPS PRIMARY LOST", "#ffff00", () => {
+                    this.addTypeTwoMessage("GPS PRIMARY LOST", "yellow", () => {
```
Comment (beheh) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js:341]: See another review comment above. Messages are never yellow.

```suggestion
                    this.addTypeTwoMessage("GPS PRIMARY LOST", "red", () => {
```

**Review by beheh — CHANGES_REQUESTED** [review]
Added a source for about the colors used for scratchpad messages.

**Comment by MisterChocker** [comment]
There is really/should be no visual difference except for the yellow messages now being amber.

**Comment by MisterChocker** [comment]
waiting for #2121 to solve conflicts because of changes to color handling of message system.

**Review by Benjozork — APPROVED** [review]
LGTM

**Comment by Benjozork** [comment]
For QA: Check that MCDU pages have the right colors, and that yellow messages now appear amber.

**Comment by Noxleee** [comment]
Category: QA Tester
Name: Noxlee #7727
Date of testing: Current date 1/12/2020
Version of the sim: 1.11.6.0
PR Tested: #2127 
Object of test: Colors of MCDU

Testing technique: Test all MCDU Pages at Gate
Steps you've made to test the Pull Request:
1. Load up AC at Gate with flight plan prefilled (from menu)
2. Checked former yellow messages - Now amber
3. Checked all pages for Cyan (I went through (EVERY!) Page of the MCDU) -> Can't find any issue / deviation
4. Checked green on PERF PAGE TO RWY FIELD -> Runway is now green
5. Checked green on PROG PAGE ECON CRZ -> ECON CRZ is now green
6. Checked missing sub row on airways page -> Filled complete Page, VIA and TO on every Row
7. Checked new flight phases to PROG PAGE -> No idea how to check this, but CRZ Flight level showed only after at take-off (Set TO thrust), do not know if there were made changes


Overall Rating: Good
Conclusions of the testing: I checked everything and did not find any issue but am not sure with the 7th check. I could not find any red in the MCDU. If there should be something red, I did not see it, or it was not red.

**Comment by MisterChocker** [comment]
Sounds great except for the CRZ FL on PROG PAGE, the behavior described by you is the behavior before I made the changes to it.
So you could actually go ahead, pull the latest version of this pr and see if this changed. (It should)

**Comment by Noxleee** [comment]
Okay, I will pull the latest version, test it and update my report later today!

**Comment by Noxleee** [comment]
> 
> 
> Category: QA Tester
> Name: Noxlee #7727
> Date of testing: Current date 1/12/2020
> Version of the sim: 1.11.6.0
> PR Tested: #2127
> Object of test: Colors of MCDU
> 
> Testing technique: Test all MCDU Pages at Gate
> Steps you've made to test the Pull Request:
> 
>     1. Load up AC at Gate with flight plan prefilled (from menu)
> 
>     2. Checked former yellow messages - Now amber
> 
>     3. Checked all pages for Cyan (I went through (EVERY!) Page of the MCDU) -> Can't find any issue / deviation
> 
>     4. Checked green on PERF PAGE TO RWY FIELD -> Runway is now green
> 
>     5. Checked green on PROG PAGE ECON CRZ -> ECON CRZ is now green
> 
>     6. Checked missing sub row on airways page -> Filled complete Page, VIA and TO on every Row
> 
>     7. Checked new flight phases to PROG PAGE -> No idea how to check this, but CRZ Flight level showed only after at take-off (Set TO thrust), do not know if there were made changes
> 
> 
> Overall Rating: Good
> Conclusions of the testing: I checked everything and did not find any issue but am not sure with the 7th check. I could not find any red in the MCDU. If there should be something red, I did not see it, or it was not red.

Here is the update:
7. Checked new flight phases to PROG PAGE -> CRZ flight level showed during pre-flight and taxi.

Overall Rating: Good
Conclusions of the testing: I tested the updated PR again, but only point 7. Now it is working correctly.
LGTM!

**Comment by Benjozork** [comment]
Should be good

**Pull request merged**